Ginger/GingerCore/ALM/Qtest/ExportToQtest.cs

```diff
@@ -0,0 +1,594 @@
+                                            
+                                            if (!System.IO.File.Exists(zipFileName))
+                                            {
+                                                ZipFile.CreateFromDirectory(activGroup.TempReportFolder, zipFileName);
+                                            }
+                                            else
+                                            {
+                                                System.IO.File.Delete(zipFileName);
+                                                ZipFile.CreateFromDirectory(activGroup.TempReportFolder, zipFileName);
+                                            }
+                                            System.IO.Directory.Delete(activGroup.TempReportFolder, true);
+                                            //Creating the Zip file - finish
+                                            //Attaching Zip file - start
+                                            AttachmentFactory attachmentFactory = (AttachmentFactory)run.Attachments;
+                                            TDAPIOLELib.Attachment attachment = (TDAPIOLELib.Attachment)attachmentFactory.AddItem(System.DBNull.Value);
+                                            attachment.Description = "TC Ginger Execution HTML Report";
+                                            attachment.Type = 1;
+                                            attachment.FileName = zipFileName;
+                                            attachment.Post();
+
+                                            //Attaching Zip file - finish
+                                            System.IO.File.Delete(zipFileName);
+                                        }
+                                    }
+
+
+                                    //create run with activities as steps
+                                    run.CopyDesignSteps();
+                                    run.Post();
+                                    StepFactory stepF = run.StepFactory;
+                                    List stepsList = stepF.NewList("");
+
+                                    //int i = 0;
+                                    int index = 1;
+                                    foreach (Step step in stepsList)
+                                    {
+                                        //search for matching activity based on ID and not order, un matching steps need to be left as No Run
+                                        int stepDesignID = (stepsList[index]).Field("ST_DESSTEP_ID");
+                                        Activity matchingActivity = activities.Where(x => x.ExternalID == stepDesignID.ToString()).FirstOrDefault();
+                                        if (matchingActivity != null)
+                                        {
+                                            switch(matchingActivity.Status)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)